loki/update.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Update Loki mixins from upstream
+#
+
+set -e
+
+TAG="helm-loki-5.47.2"
```

Comment:
Would there be a way to have this configurable ? Something like `TAG="helm-loki-$version"` so that we can alwys use the latest veresion

Reply:
That's my idea eventually but I'm no bash expert so I'm waiting for help